bambi/models.py

```diff
@@ -1000,8 +1005,11 @@ def _get_group_specific_groups(self):
         return groups
 
     def __str__(self):
-        priors = ""
         priors_common = [f"    {t.name} ~ {t.prior}" for t in self.common_terms.values()]
+        if self.intercept_term:
+            term = self.intercept_term
+            priors_common = [f"    {term.name} ~ {term.prior}"] + priors_common
```

Comment:
Why so many spaces in the front of the string?

Reply:
They're the indentation that is shown when we print the model. Like below

```
Formula: numclaims ~ offset(np.log(exposure))
Family name: Poisson
Link: log
Observations: 67856
Priors:
  Common-level effects
    Intercept ~ Normal(mu: 0, sigma: 2.5)

  Offset effects
    offset(np.log(exposure)) ~ 1
```

maybe there's a better way to do it that I'm missing

Reply:
this is fine, I was just curious!